src/actionHandlers/onMorseTranslate/const.ts

```diff
@@ -1,13 +1,13 @@
-import { ALPHABETICAL_CHARS, DECIMAL_NUMBERS } from '@utils/const';
+import { ALPHABETICAL_CHARS, DECIMAL_NUMBERS, PUNCTUATION_MARKS } from '@utils/const';
 
 export const morseChars =
-  '.- -... -.-. -.. . ..-. --. .... .. .--- -.- .-.. -- -. --- .--. --.- .-. ... - ..- ...- .-- -..- -.-- --.. .---- ..--- ...-- ....- ..... -.... --... ---.. ----. ----- /' as const;
+  '.- -... -.-. -.. . ..-. --. .... .. .--- -.- .-.. -- -. --- .--. --.- .-. ... - ..- ...- .-- -..- -.-- --.. .---- ..--- ...-- ....- ..... -.... --... ---.. ----. -----' as const;
+export const morsePunctMarks =
+  '.-... .----. .--.-. -.--.- -.--. ---... --..-- -...- -.-.-- .-.-.- -....- -..- .-.-. .-..-. ..--.. -..-.' as const;
```

Comment:
Why did u declare these two constants here and not in consts.ts file in the utils?

Reply:
> Why did u declare these two constants here and not in consts.ts file in the utils?

Because they're just related to the morse translator feature, it's not likely to use them in any other place.

Reply:
This is because the morse chars aren't that global, But the alphabetical letters, the numbers, and the punctuation are global and can be used app-wide.